tasks/2_convert-to-f/convert-to-f.js

```diff
@@ -0,0 +1,4 @@
+export const convertToF = (celsius) =>{
+  const fahrenheit = (celsius * 9/5) + 32;
```

Comment:
FYI: keep in mind that round-error could happen here https://floating-point-gui.de/

Reply:
Refactored